src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -43,10 +47,35 @@ interface ImplicitParam<T, K extends keyof T> {
   value: T[K]
 }
 
+interface ComputedParam<T, K extends keyof T> {
+  name: string
+  computeValue: (otherFields: Partial<T>) => T[K]
+}
+
 type AccountNumberSchema = {
   [Property in keyof FiatAccountSchemas[FiatAccountSchema.AccountNumber]]:
     | FormFieldParam
     | ImplicitParam<FiatAccountSchemas[FiatAccountSchema.AccountNumber], Property>
+    | ComputedParam<FiatAccountSchemas[FiatAccountSchema.AccountNumber], Property>
+}
+
+export function getAccountName({
+  accountNumber,
+  institutionName,
+}: {
```

Comment:
wonder if this should just take in fiat account schema and map it to an account name. I think we can remove the conditional because both these are required in `AccountNumber` schema. 
Could also refactor this in a follow up when the wallet starts supporting multiple schemas.

Reply:
I moved this to be an anonymous function again so it's more clear it's specific to AccountNumber